config/samples/ingressnodefirewall-demo-2.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     do-node-ingress-firewall: 'true'
   ingress:
   - sourceCIDRs:
-       - 172.20.0.0/24
+       - 172.0.0.0/8
```

Comment:
nit: 172.16.0.0/12 to cover the entire type B block instead of public addresses?